content/tutorials/use-simple-xrp-payments/send-xrp.md

```diff
@@ -238,6 +270,7 @@ To know for sure what a transaction did, you must look up the outcome of the tra
 
 - **JavaScript:** Use the [`getTransaction()` method](rippleapi-reference.html#gettransaction) to check the status of a transaction.
 - **Python:** The response of [xrpl.transaction.get_transaction_from_hash() method](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.get_transaction_from_hash) contains the results if the transaction has been validated by consensus. (See [tx result](https://xrpl.org/tx.html#response-format) for a detailed reference of the fields this can contain.)
+- **Java:** Use the `XrplClient.transaction()` method to check the status of a transaction.
```

Comment:
It's better to link every instance of the method name rather than just the first one as you appear to have done. People skim and can't always easily find the first instance.

You can use a Markdown reference link to define the URL once and refer to it multiple times (before or after). For example:

```
Use the [`XrplClient.transaction()` method][] to check the status of a transaction.

[`XrplClient.transaction()` method]: https://javadoc.io/doc/org.xrpl/xrpl4j-client/latest/org/xrpl/xrpl4j/client/XrplClient.html#transaction(org.xrpl.xrpl4j.model.client.transactions.TransactionRequestParams,java.lang.Class)
```

Though, as a warning, the Markdown parser Dactyl uses had some bugs about having `code font` in reference link text in the past. I think it might be fixed now but I'm not 100% sure so I tend to avoid using code font in reference links just to be safe.